### ISSUE TITLE
Wrong annotation for visibility feature used

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#582]
**Reported by dietmarw on 29 Jun 2011 11:26 UTC**
The following files contain an unknown annotation **`Invisible`**:
```
 Modelica/Math/package.mo
   12228:  Invisible=true,

 Modelica/Constants.mo
   102:    Invisible=true,

 Modelica/SIunits.mo
   1853:    Invisible=true,
```

I'm not quite sure if what the creator actually wanted to use is **`visible=false`** instead or if the wanted functionality is entirely different in which case `Invisible` needs to be specified. 

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/582

**Comment by modelica-trac-importer** [comment]
**Modified by dietmarw on 22 Sep 2011 05:38 UTC**

**Comment by modelica-trac-importer** [comment]
**Comment by dietmarw on 12 Mar 2012 13:15 UTC**
Fixed by removing the annotations in aae2deb66ad02bd888186d928e7f4f7eb8284f15.
